images/pytorch-pods/launch_k8s_workers.py

```diff
@@ -190,8 +203,32 @@ def main(argv):
               'cpu': FLAGS.cpu,
               'memory': FLAGS.memory,
             }
-          }
-        }]
+          },
+          'volumeMounts': [
+            {
+              'name': 'dshm',
+              'mountPath': '/dev/shm/',
+            },
+            *[
+              {
+                'name': name,
+                'mountPath': mount_path,
+              } for name, mount_path in volumes.items()
+            ],
+          ],
+        }],
+        'volumes': [
+          {
+            'name': 'dshm',
+            'emptyDir': { 'medium': 'Memory' },
+          },
+          *[
+            {
+              'name': name,
+              'persistentVolumeClaim': { 'claimName': name }
+            } for name, _ in volumes.items()
```

Comment:
Oh here would be probably be a bad spot to have that empty list mentioned in my other comment

Reply:
Fixed.